src/verilog/verilog_typecheck_expr.h

```diff
@@ -200,6 +200,7 @@ class verilog_typecheck_exprt:public verilog_typecheck_baset
   }
 
   void require_sva_sequence(const exprt &) const;
+  void require_sva_property(exprt &);
```

Comment:
It's a bit weird that only the latter has a side effect.

Reply:
Coming as a separate PR.

Reply:
#1047